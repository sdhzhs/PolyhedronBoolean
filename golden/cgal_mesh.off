OFF
12 14 0

2 2 1.343
2 2 2
4 2 2
4 2 1.343
4 4 1.343
2 4 1.343
2.5 3.5 1.343
3.5 3.5 1.343
3.5 2.5 1.343
2.5 2.5 1.343
2 4 2
4 4 2
4  1 0 3 2
3  5 9 0
3  6 9 5
3  5 7 6
3  5 4 7
3  7 4 3
3  7 3 8
3  8 3 0
3  8 0 9
4  5 0 1 10
4  10 1 2 11
4  2 3 4 11
4  4 5 10 11
4  8 9 6 7

