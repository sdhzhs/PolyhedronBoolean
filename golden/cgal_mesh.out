8, 6
2.000000000000000e+00 2.000000000000000e+00 2.000000000000000e+00
2.000000000000000e+00 2.000000000000000e+00 1.343000000000000e+00
4.000000000000000e+00 2.000000000000000e+00 1.343000000000000e+00
4.000000000000000e+00 2.000000000000000e+00 2.000000000000000e+00
2.000000000000000e+00 4.000000000000000e+00 1.343000000000000e+00
4.000000000000000e+00 4.000000000000000e+00 1.343000000000000e+00
2.000000000000000e+00 4.000000000000000e+00 2.000000000000000e+00
4.000000000000000e+00 4.000000000000000e+00 2.000000000000000e+00
0 1 2 3 
2 1 4 5 
4 1 0 6 
6 0 3 7 
3 2 5 7 
5 4 6 7 
